packages/synapse-react-client/src/synapse-queries/featureflags/useGetFeatureFlags.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import SynapseClient from '../../synapse-client'
+import { SynapseClientError, useSynapseContext } from '../../utils'
+import { FeatureFlags, FeatureFlagEnum } from '@sage-bionetworks/synapse-types'
+
+export function useGetFeatureFlags(
```

Comment:
Rename if this hook only provides one flag value
```suggestion
export function useGetFeatureFlag(
```